Azure/azure-cli
# [AppService] Add support for deployment logs for webapp in CLI

**Description** 

closes #13734 
Add support for new webapp commands for deployment logs: `az webapp log deployment show` and `az webapp log deployment list`. 

**Testing Guide**  

Create & deploy a web or function app

Run `az webapp log deployment list -g ResourceGroup -n WebappName`. This should give you the same list of deployment logs you see when you go to `https://<webapp>.scm.azurewebsites.net/api/deployments`

Run `az webapp log deployment show -g ResourceGroup -n WebappName`. This should give you the latest deployment log

Copy the Deployment ID from any deployment

Run `az webapp log deployment show -g ResourceGroup -n WebappName --deployment-id DeploymentId`. This should give you the log of that particular deployment.

**History Notes**  

[App Service] `az webapp log deployment show`: Show the latest deployment log, or the deployment logs of a specific deployment if deployment-id is specified
[App Service] `az webapp log deployment list`: List of deployment logs available

---

This checklist is used to make sure that common guidelines for a pull request are followed.
